frontend/src/components/Footer/Footer.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Footer from './Footer';
+
+import { render } from '@testing-library/react';
+import "@testing-library/jest-dom/extend-expect"; 
+
+it("renders without crashing", ()=> {
+    const div = document.createElement("div");
+    ReactDOM.render(<Footer />, div);
+});
+
+it("renders Footer correctly", ()=> {
+    const {getByTestId} = render(<Footer text="Copyright © TourDeFoos 2021"/>)
```

Comment:
Nie musisz tak "wyciągać" z komponentu funkcji. Szczególnie przy większej liczbie asercji to może być skomplikowane.
Zobacz inne testy i użycie screen.